### ISSUE TITLE
Item chart with symbols: symbols are stretched

### ISSUE THREAD

**Issue by danielbartsch** [highcharts/highcharts#11451]
#### Expected behaviour
symbols should look like initially

#### Actual behaviour
symbols are stretched if hidden and then shown again.

#### Live demo with steps to reproduce
appears in the original live demo link in https://github.com/highcharts/highcharts/blob/master/docs/item-series.md
https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/series-item/symbols/

Steps to reproduce:
![itemChartStretch](https://user-images.githubusercontent.com/19350749/61618726-b9cad000-ac6d-11e9-8657-596f01b556ea.gif)

#### Product version
latest. See the jsfiddle link

#### Affected browser(s)
tested firefox, chrome, opera. Appears in all of them.

**Comment by pawelfus** [comment]
Thanks for reporting the issue @danielbartsch 

**Workaround:**
Disable animation and apply width once again to the element: https://jsfiddle.net/BlackLabel/a8gq6p1j/ 

Snippet:
```js
(function(H) {

  H.wrap(H.seriesTypes.item.prototype, 'drawPoints', function(p) {

    p.apply(this, Array.prototype.slice.call(arguments, 1));

    this.points.forEach(function(point) {

      if (point.graphic) {
        for (var i = 0; i < point.y; i++) {
          if (point.graphics[i] && point.graphics[i].attr) {
            point.graphics[i].attr({
              width: point.graphics[i].width
            });
          }
        }
      }
    });

  });

})(Highcharts);
```

*Internal note:*
This is caused by applying width only when both, width and height, exist here:
https://github.com/highcharts/highcharts/blob/91df181bb32a3d8235d9e83ea527893d55d9df17/ts/parts/SvgRenderer.ts#L5269-L5288

In other words, calling `attr({ width: 100, height: 100})` won't apply `width`, only the second prop: `height`, will pass on the condition and modify `imgSize`.
